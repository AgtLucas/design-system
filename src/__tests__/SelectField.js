import React from 'react'
import renderer from 'react-test-renderer'
import { ThemeProvider, SelectField } from '..'

describe('SelectField', () => {
  test('renders', () => {
    const json = renderer
      .create(
        <ThemeProvider>
          <SelectField>Test</SelectField>
        </ThemeProvider>
      )
      .toJSON()
    expect(json).toMatchSnapshot()
  })

  test('renders with isSelected prop', () => {
    const json = renderer
      .create(
        <ThemeProvider>
          <SelectField isSelected>Test</SelectField>
        </ThemeProvider>
      )
      .toJSON()
    expect(json).toMatchSnapshot()
  })
})
