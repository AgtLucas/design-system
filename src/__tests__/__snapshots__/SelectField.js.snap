// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectField renders 1`] = `
.c2 {
  font-size: 12px;
}

.c1 {
  padding-left: 64px;
  padding-right: 64px;
  padding-top: 16px;
  padding-bottom: 16px;
  null: null;
}

.c1:hover {
  background-color: #007aff;
  color: #fff;
  cursor: pointer;
}

.c0 {
  font-family: 'Montserrat','Helvetica Neue',Helvetica,Arial,sans-serif;
  line-height: 1.4;
}

.c0 * {
  box-sizing: border-box;
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <div
      className="c2"
      fontSize={0}
    >
      Test
    </div>
  </div>
</div>
`;

exports[`SelectField renders with isSelected prop 1`] = `
.c2 {
  font-size: 12px;
}

.c1 {
  padding-left: 64px;
  padding-right: 64px;
  padding-top: 16px;
  padding-bottom: 16px;
  background-color: #007aff;
  color: #fff;
}

.c1:hover {
  background-color: #007aff;
  color: #fff;
  cursor: pointer;
}

.c0 {
  font-family: 'Montserrat','Helvetica Neue',Helvetica,Arial,sans-serif;
  line-height: 1.4;
}

.c0 * {
  box-sizing: border-box;
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <div
      className="c2"
      fontSize={0}
    >
      Test
    </div>
  </div>
</div>
`;
